text/0000-docs-information-architecture

```diff
@@ -0,0 +1,154 @@
+- Start Date: 2018-04-25
+- RFC PR:
+- Gatsby Issue:
+
+# Summary
+
+A proposal to add a second (maybe even third) level of navigation to the docs on gatsbyjs.org and to add new top-level categories.
+
+# Motivation
+
+Through many user research interviews, I've heard that it's difficult for people to find what they are looking for in the docs. They have mentioned many instances where the information they are looking for does not exist, it exists but they couldn't find it, or it exists and they could find it in multiple places (not in one single place). This is likely because the docs have outgrown their current categories.
+This design also proposes a possibly heuristic for determining what we can reasonably commit to maintaining as part of Gatsby core docs, and what might be too far outside the scope of core docs (either because the topics don't fit or because they are not a priority, given our limited time and resources). includes a list of what will *not* be included in Gatsby core docs.
+
+# Detailed design
+
+Here is a possible restructure of the current docs information architecture (with the inclusion of many docs we still need to write):
+
+Tutorials
+- Quick Start
+- Build and deploy a site
+    - Introduction
+    - 0. Setting up the development environment
+    - 1. Creating pages
+    - 2. Adding styles
+    - 3. Using layouts to build reusable site sections
+    - 4. Deploying a site live online (debating whether to keep this inside part 1, where it currently is)
+- Build a blog with GraphQL
+    - 5. Querying data for your blog
+    - 6. Source plugins and rendering queried data
+    - 7. Transformer plugins and transforming data into site content
+    - 8. Programmatically creating pages from data
+- Adding images to your site (or "Adding svg or blur-up effect to images)*
+- Creating your own plugin (tutorial for creating a plugin we've already vetted as easy to create)*
+- E-commerce*
+- User authentication*
+
+Reference docs
+- How Gatsby works
+    - The Gatsby core philosophy*
+    - Basic structure of a Gatsby site (includes config.js)*
+    - How Gatsby works with GraphQL (includes Node interface and how GraphQL works)*
+    - How Gatsby works with React (API specification)*
+    - How Gatsby works with Redux (bound action creators)*
+    - How Gatsby works with plugins
+    - PRPL pattern
+    - Lifecycle APIs
+- API references
+    - @jlengstorf this idea is from you, don't know how to fill this in...*
+    - API specification
+    - Gatsby Browser APIs
+    - Gatsby Node APIs
+    - Gatsby SSR APIs 
+
+- Gatsby CLI commands
```

Comment:
perhaps this should be moved to "setting up your dev environment?"

Reply:
Oh, great point. Yes, it ought to be easy to access from part 0 of the tutorial. I think it also belongs here since it would be written like a reference document (just like the index in the back of a book).